extensions/default/src/DicomWebProxyDataSource/index.js

```diff
@@ -0,0 +1,74 @@
+import { IWebApiDataSource } from '@ohif/core';
+import { createDicomWebApi } from '../DicomWebDataSource/index';
+
+function createDicomWebProxyApi(
```

Comment:
Can you add couple of lines in the documentation of this function and also explain the schema for which the data source is being used for? what does the url look like as an exmaple etc.

Reply:
For instance, it seems like it needs `servers` info to be present at the url

Reply:
I added a short comment to the class, but the (added) markdown documentation is probably a better way to see this